components/sections/introduction/content.ja-JP.md

```diff
@@ -0,0 +1,10 @@
+# バッとした導入
+
+まずは良い例から始めましょう。ベジエ曲線というのは、下の図に表示されているもののことです。ベジエ曲線はある始点からある終点へと延びており、その曲率は1個以上の「中間」制御点に左右されています。さて、このページの図はどれもインタラクティブになっていますので、ここで曲線をちょっと操作してみましょう。点をドラッグしたとき、曲線の形がそれに応じてどう変化するのか、確かめてみてください。
+
+<div className="figure">
+  <Graphic inline={true} title="2次のベジエ曲線" setup={ this.drawQuadratic } draw={ this.drawCurve }/>
+  <Graphic inline={true} title="3次のベジエ曲線" setup={ this.drawCubic } draw={ this.drawCurve }/>
+</div>
+
+ベジエ曲線は、CAD（computer aided design）や CAM（computer aided manufacturing）のアプリケーションで多用されています。もちろん、Adobe Illustrator・Photoshop・Inkscape・Gimp などのグラフィックデザインアプリケーションや、SVG（scalable vector graphics）・OpenType フォント（otf/ttf）のようなグラフィック技術でも利用されています。ベジエ曲線はたくさんのものに使われていますので、これについてもっと詳しく学びたいのであれば……さあ、準備しましょう！
```

Comment:
I have not decided yet whether we put in a space between alphanumeric and Japanese characters. In terms of typography, spacing of the Japanese (fullwidth) punctuations are also problematic. The [`text-spacing` propterty](https://drafts.csswg.org/css-text-4/#text-spacing-property) in the CSS Text Module Level 4 should solve them, but currently no browsers support it.

Anyway, I would like to give priority to translations for now, rather than these formatting issues.